interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -1599,8 +1600,12 @@ namespace cling {
     namespace internal {
       Value EvaluateDynamicExpression(Interpreter* interp, DynamicExprInfo* DEI,
                                       clang::DeclContext* DC) {
-        Value ret = interp->Evaluate(DEI->getExpr(), DC,
-                                     DEI->isValuePrinterRequested());
+        Value ret;
```

Comment:
Does default init of Value cost in run-time?  If so, you might consider using a lamda function instead of a scope.

Reply:
Nice one, thanks! I've applied your idea.